content/en/docs/collector/builder.md

```diff
@@ -0,0 +1,107 @@
+---
+title: Builder
+weight: 25
+---
+
+The OpenTelemetry Collector Builder (ocb) is a command-line interface tool that
+generates sources and binaries for OpenTelemetry Collector
+[distributions](../distributions).
+
+This tool is currently being used to build both the core and the contrib
+distributions, and can be used to build custom distributions as well, allowing
+users to cherry-pick which components they need for their production usage,
+potentially even including their own custom components.
+
+## Installing
+
+You can download `ocb` from the OpenTelemetry Collector's
+[release page](https://github.com/open-telemetry/opentelemetry-collector/releases)
+on GitHub. A container image is **NOT** available at this time. Prefer to use
+the same version number that you'll use in your manifest, as the builder is only
+guaranteed to work with the OpenTelemetry Collector API matching the builder's
+version.
+
+## Manifest
+
+The central part of a custom distribution is the distribution's manifest. This
+manifest defines the metadata for the distribution and lists all components that
+should be included.
+
+This is an example `manifest.yaml`, containing only a couple of components:
+
+```yaml
+dist:
+  module: github.com/open-telemetry/opentelemetry-collector-releases/sidecar
+  name: sidecar
+  description: OpenTelemetry Collector Sidecar version
+  version: 0.73.0
+  output_path: ./_build
+  otelcol_version: 0.73.0
```

Comment:
Here and elsewhere in the document, replace the current version (0.73.0) by `{{% param collectorVersion %}}`:

```suggestion
  otelcol_version: {{% param collectorVersion %}}
```

Reply:
I had to revert this, as it was causing the prettier script to break the final snippet in places where it would render the final YAML as an invalid config. Is there a way to bypass the prettier for this snippet?

Reply:
Yes, place `<!-- prettier-ignore -->` before the start of the code block.

If you prefer, I can make this change after the PR is merged. Let me know.